backend/uclapi/uclapi/settings.py

```diff
@@ -154,9 +153,16 @@
 
 DATABASE_ROUTERS = ['uclapi.dbrouters.ModelRouter']
 
-RAVEN_CONFIG = {
-    'dsn': os.environ.get("SENTRY_DSN"),
-}
+if os.environ.get('SENTRY_DSN'):
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+
+    sentry_sdk.init(
+        dsn=os.environ.get('SENTRY_DSN'),
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=0.01,
+        send_default_pii=True
```

Comment:
does this work for us? We don't use the default Django authentication stuff.

Reply:
We don't get the Django auth benefit however this sends us additional information about the user client (such as ip address and user agent). In the interest of privacy I have disabled it.